src/Level.h

```diff
@@ -0,0 +1,27 @@
+/*****************************************************************************
+ * Copyright (c) 2018-2021 openblack developers
+ *
+ * For a complete list of all authors, please refer to contributors.md
+ * Interested in contributing? Visit https://github.com/openblack/openblack
+ *
+ * openblack is licensed under the GNU General Public License version 3.
+ *****************************************************************************/
+
+#pragma once
+
+#include <string>
+#include <filesystem>
+
+namespace openblack
+{
+class Level
+{
+public:
+	Level(std::filesystem::path path) : _name(path.filename().string()), _scriptPath(path) {}
+	const std::string& GetName() const { return _name; };
```

Comment:
This could be just `_scriptPath.filename().string()` instead of keeping a copy

Reply:
Good idea